src/main/java/frc/robot/subsystems/funnel/FunnelState.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.subsystems.funnel;
+
+public enum FunnelState {
+
+	SPEAKER(0.6),
+	AMP(-0.6),
+	OUTTAKE(0.4),
+	NOTE_TO_SHOOTER(0.5),
```

Comment:
why u are not using NOTE_TO_SHOOTER in ELEVATOR_TO_SHOOTER,
if u need different states find more informative name

Reply:
ur right, only the intake roller needs a different state